.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/
-vendor/
\ No newline at end of file
+vendor/
+composer.lock
```

Comment:
How comes that we won't use a composer.lock here?

Reply:
Leaving out composer.lock for libraries allows the CI setup to test against multiple backwards-incompatible versions of the dependencies. That way we can add more supported versions of PHP without getting test failures if we need to upgrade a dependency for that specific version.

It's ignored when anyone uses this library anyway, so won't impact consumers

Reply:
Ah I see, alright :+1: